src/styles/BookForm.css

```diff
@@ -1,11 +1,53 @@
 .containerCreateBooks {
   width: 90%;
-  height: 100px;
+  height: 150px;
   margin: 80px auto;
   padding: 20px 0;
+  padding-bottom: 60px;
+  margin-top: 10px;
 }
 
 form {
+  font-family: 'Montserrat', sans-serif;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+}
+
+.line {
+  height: 0.125rem;
+  margin-top: 10px;
+  margin-bottom: 30px;
+  border: solid 1px #ccc;
+}
+
+h3 {
+  width: 11rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-size: 1.25rem;
+  font-weight: bold;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.add-btn {
+  width: 11.5rem;
+  height: 2.813rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+  color: #fff;
+}
+
+input,
+select[option] {
+  height: 1.188rem;
+  font-size: 1rem;
+  border: solid 1px #ccc;
+}
+
```

Comment:
- [ ] Great job so far styling your form input nicely. However, I think you can improve the looks of it by adding some `padding` to it so input text is separated from the borders of the input. Kindly consider making these adjustments as it would help create a good user experience and help make your web app look more professional.

- [ ] Also, consider making sure that your `input` border colors match that of the given design as they look darker compared to that of the given design. See below for more details:

<table>
<tr>
<th align=“middle”>Original design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="864" alt="Screenshot 2023-08-24 at 22 22 19" src="https://github.com/Bhekisile/Bookstore/assets/107192076/38cdd208-c681-482a-b69c-bef785ed5c5a">
</td>
<td>
<img width="1298" alt="Screenshot 2023-08-24 at 22 21 56" src="https://github.com/Bhekisile/Bookstore/assets/107192076/a1793b08-6fdc-42b7-9be6-6c2db9f9e6fb">
</td>
</tr>
</table>